src/Interfaces.ts

```diff
@@ -15,10 +16,17 @@ export interface IResolverValidationOptions {
     requireResolversForAllFields?: boolean;
 }
 
+export interface IResolverOptions {
+    resolve?: GraphQLFieldResolveFn;
+    __resolveType?: IResolveTypeFn;
+    __isTypeOf?: IIsTypeOfFn;
```

Comment:
i've added deprecated-decorator,
then it emits:
Function `attachConnectorsToContext` has been deprecated since version 0.7.0.
    at /home/hagai/a2z/gits/infastructure/graphql-tools/src/schemaGenerator.ts:35:51

you can enter travis logs and view the results of the tests.